bin/container-entrypoint.d/ems-setup.sh

```diff
@@ -71,9 +89,21 @@ if [ \${1:-list} = sql ] || [ \${1:-list} = dump ] ; then
 else
   php -d memory_limit=\${CLI_PHP_MEMORY_LIMIT:-512M} /opt/src/bin/console \$@
 fi;
+EOL
+
+  cat >/opt/bin/ems-jobs/$_instance_name <<EOL
+#!/bin/bash
+
+echo ---
+echo Running ems-jobs for [ $_instance_name ]
+echo ---
+
+/opt/bin/$_instance_name -vvv ems:job:run
```

Comment:
Not convinced by the -vvv it will generate a lot of logs. For a first version ok, but it would be better to parametrize it

Reply:
About frequency, the documentation say that only Supervisor events ```TICK_5```, ```TICK_60``` and ```TICK_3600``` are availables.  I think it is acceptable and sufficient for ems-jobs feature that must be part of global solution.

Reply:
About verbosity, I suggest to add environment variable prefix ```EMS_JOBS_``` for all envvars which configure EMS Jobs feature.  I think the first must be ```EMS_JOBS_ENABLE``` to configure the supervisor event and use it instead of interactive web page.  After, we can add ```EMS_JOBS_VERBOSE``` and ```EMS_JOBS_OPTS```

```
/opt/bin/$_instance_name ${EMS_JOBS_VERBOSE} ems:job:run ${EMS_JOBS_OPTS}
```

Reply:
To avoid confusion with EMS Common Bundle configuration, it is better to use non EMS_ prefixed variables.
```JOBS_ENABLE``` and ```JOBS_OPTS``` are used in this purpose.  Remove JOBS_VERBOSE which is content can be included in JOBS_OPTS variable content.